src/bsoncxx/include/bsoncxx/v_noabi/bsoncxx/stdx/type_traits.hpp

```diff
@@ -0,0 +1,430 @@
+#pragma once
+
+#include <type_traits>
+
+#include <bsoncxx/config/prelude.hpp>
+
+namespace bsoncxx {
+
+BSONCXX_INLINE_NAMESPACE_BEGIN
```

Comment:
```suggestion
inline namespace v_noabi {
```
Consistency with changes in https://github.com/mongodb/mongo-cxx-driver/pull/1039.

Reply:
Could that macro be removed completely, or is it too much part of the public API?